syntax = "proto3";

option go_package = "github.com/JonasKaad/ScalableDataIntegration/GeneratedClients/go/parser";

package sdi.parser;

service Parser {
	rpc ParseCall (ParseRequest) returns (ParseResponse);
}

message ParseRequest {
	bytes raw_data = 1;
	string format = 2;
}

message ParseResponse {
	bool success = 1;
	optional string err_msg = 2;
}
